### ISSUE TITLE
C.67: A polymorphic class should suppress copying

### ISSUE THREAD

**Issue by gennaroprota** [isocpp/CppCoreGuidelines#1543]
_C.67: A polymorphic class should suppress copying_ is IMHO a bit off for two reasons:

- it doesn't consider exception classes, which *must* be copy constructible

- "suppress copying" may make the reader think they should explicitly disable copying, but that's not necessary if the base class is already non-copyable

So, I suggest rewording the guideline to:

  "A polymorphic, non-exception, class should not be copyable"

**Comment by hsutter** [comment]
Editors call: Thanks, we think exception hierarchies already addressed in the text of C.67 since #1246 and that seems to capture it.

**Comment by cubbimew** [comment]
note the second bullet point (about explicitly disabled copying) is now #1635
